Resources/Prototypes/Entities/janitor.yml

```diff
@@ -104,3 +104,80 @@
   - type: Item
     sprite: Objects/Specific/Janitorial/wet_floor_sign.rsi
 
+- type: entity
+  name: soap
+  id: Soap
+  parent: BaseItem
+  description: A cheap bar of soap. Doesn't smell.
+  components:
+    - type: Sprite
+      sprite: Objects/Specific/Janitorial/soap.rsi
+      state: soap
+
+    - type: Icon
+      sprite: Objects/Specific/Janitorial/soap.rsi
+      state: soap
+
+    - type: Item
+      sprite: Objects/Specific/Janitorial/soap.rsi
+
+    - type: Slippery
+      paralyzeTime: 2.5
+
+- type: entity
+  name: soap
+  id: SoapNT
+  parent: BaseItem
```

Comment:
why isn't this just `Soap`? ie: SoapNT, SoapWafflCo and SoapSyndie should just be `parent: Soap`

Reply:
Ah, totally. I mindlessly copypasted soap and forgot to do that-- Fixed now, thanks